README.md

```diff
@@ -0,0 +1,7 @@
+# Important
+
+Ensure you update the version number in `src/<plugin>/BUILD` before any merge to `main`. Failure to do so will overwrite the existing, published version.
```

Comment:
PyPI actually won't let you overwrite a published package. What would happen is that the pipeline `put` step would fail due to the version conflict.